build/IPDK_Container/Ubuntu20.04/Dockerfile

```diff
@@ -0,0 +1,76 @@
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+#
+# Version 0.1.0
+
+ARG OS_VERSION=${OS_VERSION}
+
+FROM ubuntu:${OS_VERSION} AS base
+LABEL maintainer="neha1.singh@intel.com"
+ARG PROXY
+
+SHELL ["/bin/bash", "-c"]
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Installing dependent packages required for DPDK Build
+RUN apt update && apt install -y apt-utils
+
+RUN apt install -y build-essential git sudo
+
+RUN apt -y install meson && \
+  apt -y install cmake && \
+  apt -y install gdb && \
+  apt -y install libtool && \
+  apt -y install autoconf && \
+  apt -y install autoconf-archive && \
+  apt -y install automake && \
+  apt -y install scapy && \
+  apt -y install iproute2 && \
+  apt -y install net-tools && \
+  apt -y install libgflags-dev && \
+  apt -y install iputils-ping && \
+  apt -y install libcctz2 && \
+  apt -y install libjson-c-dev && \
+  apt -y install libffi-dev && \
+  apt -y install libssl-dev && \
+  apt -y install python && \
+  apt -y install python3-cffi && \
+  apt -y install python3-pip && \
+  apt -y install libedit-dev && \
+  apt -y install libexpat1-dev && \
+  apt -y install pip && \
+  apt -y install flex && \
+  apt -y install bison && \
+  apt -y install libgc-dev && \
+  apt -y install vim && \
+  apt -y clean all
```

Comment:
`apt` is designed as an end-user command-line tool and it has the following warning message:
> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

`apt-get` is prefered for use in scripts for backward compatibility

In addition, it would be better to run a single command to install all packages instead of running a separate install command for each package. For example:
```

RUN apt-get -y install \
    build-essential \
    git \
    sudo \
    meson \
    cmake \
    gdb \
    libtool \
    autoconf \
    autoconf-archive \
    automake \
    scapy \
    iproute2 \
    net-tools \
    libgflags-dev \
    iputils-ping \
    libcctz2 \
    libjson-c-dev \
    libffi-dev \
    libssl-dev \
    python \
    python3-cffi \
    python3-pip \
    libedit-dev \
    libexpat1-dev \
    pip \
    flex \
    bison \
    libgc-dev \
    vim && \
    apt-get -y clean all
```

Reply:
fixed it in latest push.